### DISCUSSION TITLE
Next version of Refprop

### DISCUSSION THREAD

**Discussion by inamdarharshad** [usnistgov/REFPROP-issues#604] [General]
Hi Ian,

When can we expect the next version (v10.1) of Refprop to be released? 

Will this next version include interaction parameters for binary pairs measured under the DOE / DOD-sponsored projects?
I mean the interaction parameters published in your article in the _Journal of Physical and Chemical Reference Data_ in March 2023?  
(https://doi.org/10.1063/5.0135368) 
 
Will this next version include mixture files (and interaction parameters) for blends designated by ASHRAE 34 since v10.0 was released? 

Thank you,
Harshad

**Comment by ianhbell** [comment]
I am afraid that I cannot provide a timeline for v10.1 as that is out of my hands. Furthermore, there is a bug in REFPROP 10.0 that is keeping me from providing the large HMX.BNC with all the mixture models that I developed, otherwise I would have done that a long time ago. Another person asked about something similar recently, so I might do a temporary fix of providing a file for "refrigerant" interaction parameters that would not hit the bug and could be used in version 10.0, but it would not include all the mixture models that we have developed.

**Comment by inamdarharshad** [comment]
I understand, Ian. Thank you for the status update!
